conans/client/build/cmake.py

```diff
@@ -15,14 +16,26 @@
 from conans.client.tools.env import environment_append, _environment_add
 from conans.client.tools.oss import cpu_count, args_to_string
 from conans.errors import ConanException
-from conans.model.conan_file import ConanFile
 from conans.model.version import Version
 from conans.util.config_parser import get_bool_from_text
 from conans.util.files import mkdir, get_abs_path, walk, decode_text
-from conans.util.runners import version_runner
 
 
-class CMake(object):
+class BaseCMakeBuildHelper(object):
+    """ Just a couple of functions shared between the CMake build helpers """
+
+    @staticmethod
+    def get_version():
+        try:
+            out, _ = subprocess.Popen(["cmake", "--version"], stdout=subprocess.PIPE).communicate()
```

Comment:
I think we have had a config entry for the conan_cmake_program, we might be getting the version of wrong cmake instance here

Reply:
Good point, I think this is not tested in the codebase, and it is broken.